Scripts/Python/ki/__init__.py

```diff
@@ -997,21 +997,24 @@ def OnRTChat(self, player, message, flags):
             if cFlags.broadcast and cFlags.channel != self.chatMgr.privateChatChannel:
                 return
 
-            # Is the message from an ignored plaer?
+            # Is the message from an ignored player?
             vault = ptVault()
             ignores = vault.getIgnoreListFolder()
             if ignores is not None and ignores.playerlistHasPlayer(player.getPlayerID()):
                 return
 
+            if cFlags.lockey:
+                keys = message.split(":")
+                message = PtGetLocalizedString(keys[0], [player.getPlayerName(), PtGetLocalizedString(keys[1] if len(keys) > 1 else "KI.EmoteStrings.Their")])
```

Comment:
If you had `LocKey` available here, this could be simplified to:
```suggestion
            if cFlags.lockey:
                keys = LocKey(*message.split(":"))
                message = PtGetLocalizedString(keys.message [player.getPlayerName(), PtGetLocalizedString(keys.pronoun)])
```

Reply:
I'm confused how this code is here. This is not the latest version from my commit last night...

Reply:
I made this go. Thanks for the suggestion.